docs/software/uenv/management.md

```diff
@@ -0,0 +1,364 @@
+
+    ```console
+    $ uenv image ls prgenv-gnu
+    uenv                           arch   system  id                size(MB)  date
+    prgenv-gnu/24.11:v1            gh200  daint   b81fd6ba25e88782   4,191    2024-11-27
+    prgenv-gnu/24.7:v3             gh200  daint   b50ca0d101456970   3,859    2024-08-23
+    ```
+
+    Or only uenv with the tag `v3`:
+
+    ```console
+    $ uenv image ls :v1
+    uenv                           arch   system  id                size(MB)  date
+    namd/3.0:v3                    gh200  daint   49bc65c6905eb5da   4,028    2024-12-12
+    prgenv-gnu/24.7:v3             gh200  daint   b50ca0d101456970   3,859    2024-08-23
+    ```
+
+!!! example "filtering downloaded uenv"
+
+This command will show you all images that have been downloaded to your repo, and labelled for the current cluster.
+The examples above were executed on [Daint][ref-cluster-daint], and we can see that .
+
+To view all images that are available for all clusters
+
+```console
+$ uenv image ls @*
+```
+
+[](){#ref-uenv-image-rm}
+### `uenv image rm`
+
+To remove downloaded images from your repo, use the `uenv image rm` command.
+
+!!! example "removing uenv from a repo"
+    In the following examples
+    ```console
+    $ uenv image ls
+    uenv                             arch   system  id                size(MB)  date
+    cp2k-spm-tools/1.5.0:1658742098  gh200  daint   01edd4331d3676b2     668    2025-02-06
+    eccodes/cug:1800983313           gh200  daint   cab8e857967d0708   4,036    2025-05-05
+    editors/24.7:v2                  gh200  daint   e7b0d930df729da5   1,270    2024-09-04
+    gpaw/25.1:1639708786             gh200  daint   f43e2034d7d8849f     695    2025-01-24
+    gromacs/2024:v1                  gh200  daint   b58e6406810279d5   3,658    2024-09-12
+
+    # remove cp2k-spm-tools using its label
+    $ uenv image rm cp2k-spm-tools/1.5.0:1658742098
+    the following uenv was removed:
+    cp2k-spm-tools/1.5.0:1658742098  gh200  daint   01edd4331d3676b2     668    2025-02-06
+
+    # remove gpaw using its id
+    $ uenv image rm f43e2034d7d8849f
+    the following uenv was removed:
+    gpaw/25.1:1639708786             gh200  daint   f43e2034d7d8849f     695    2025-01-24
+
+    $ uenv image ls
+    uenv                             arch   system  id                size(MB)  date
+    eccodes/cug:1800983313           gh200  daint   cab8e857967d0708   4,036    2025-05-05
+    editors/24.7:v2                  gh200  daint   e7b0d930df729da5   1,270    2024-09-04
+    gromacs/2024:v1                  gh200  daint   b58e6406810279d5   3,658    2024-09-12
+    ```
+
+[](){#ref-uenv-image-add}
+### `uenv image add`
+
+To add a uenv SquashFS file to a repo, so that it can be used with a [label][ref-uenv-labels], use the `uenv image add` command.
+
+!!! example "add an image to the default repo"
+    ```console
+    $ uenv image add myenv/24.7:v1@eiger%zen2 ./store.squashfs
```

Comment:
in this case it accepts a relative filepath? I've never used this. Good to know!